src/identifier.js

```diff
@@ -4,40 +4,40 @@ import TestTrackConfig from './testTrackConfig';
 import Visitor from './visitor';
 
 var Identifier = function(options) {
-    this.visitorId = options.visitorId;
-    this.identifierType = options.identifierType;
-    this.value = options.value;
+  this.visitorId = options.visitorId;
+  this.identifierType = options.identifierType;
+  this.value = options.value;
 
-    if (!this.visitorId) {
-        throw new Error('must provide visitorId');
-    } else if (!this.identifierType) {
-        throw new Error('must provide identifierType');
-    } else if (!this.value) {
-        throw new Error('must provide value');
-    }
+  if (!this.visitorId) {
+    throw new Error('must provide visitorId');
+  } else if (!this.identifierType) {
+    throw new Error('must provide identifierType');
+  } else if (!this.value) {
+    throw new Error('must provide value');
+  }
 };
 
-Identifier.prototype.save = function(identifierType, value) {
-    var deferred = $.Deferred();
+Identifier.prototype.save = function() {
```

Comment:
this was a broken and confusing interface. the args aren't used in the method and were never passed by any caller.